src/simple-keyring.test.ts

```diff
@@ -115,6 +125,29 @@ describe('simple-keyring', function () {
         keyring.signTransaction(notKeyringAddress, tx),
       ).rejects.toThrow('Simple Keyring - Unable to find matching address.');
     });
+
+    /**
```

Comment:
I'm assuming that the original code in `signTransaction` was designed to account for `ethereumjs-tx`? If so, this seems very similar to [åsomething I saw in the tests for `eth-ledger-bridge-keyring`](https://github.com/MetaMask/eth-ledger-bridge-keyring/blob/fe7753860f1dba0da1cdc3b8bd63eee5cfa70102/src/ledger-bridge-keyring.test.ts#L545), where there [we are bringing in `ethereumjs-tx` in addition to `@ethereumjs/tx`](https://github.com/MetaMask/eth-ledger-bridge-keyring/blob/fe7753860f1dba0da1cdc3b8bd63eee5cfa70102/src/ledger-bridge-keyring.test.ts#L8). Is that a technique we can use here as well?

Reply:
Awesome suggestion!